<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>352</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>321</y>
     <width>458</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>458</width>
     <height>32</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_WindowBorderStyle">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>292</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Borderless</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_IsWindowed">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>262</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Windowed Mode</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>0</y>
     <width>440</width>
     <height>158</height>
    </rect>
   </property>
   <property name="title">
    <string>Display Setup</string>
   </property>
   <widget class="QLabel" name="labelAspRatio">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>14</y>
      <width>140</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right;</string>
    </property>
    <property name="text">
     <string>Aspect Ratio : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelAA">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>74</y>
      <width>140</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right;</string>
    </property>
    <property name="text">
     <string>Antialiasing : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelAFiltering">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>104</y>
      <width>140</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right;</string>
    </property>
    <property name="text">
     <string>Anisotropic Filtering : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideoResolution">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>44</y>
      <width>140</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right;</string>
    </property>
    <property name="text">
     <string>Video Resolution : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxARatio">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>14</y>
      <width>280</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxVResolution">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>44</y>
      <width>280</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAA">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>74</y>
      <width>280</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAFiltering">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>104</y>
      <width>280</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>162</y>
     <width>440</width>
     <height>74</height>
    </rect>
   </property>
   <property name="title">
    <string>Level of Details</string>
   </property>
   <widget class="QPushButton" name="pushButtonVLow">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>14</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VideoReset">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>14</y>
      <width>132</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset to Defaults</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AdvSettings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>42</y>
      <width>132</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Advance Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVMed">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>14</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VHigh">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>14</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VEpic">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>14</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Epic</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
